src/Repository/FileRepository.hpp

```diff
@@ -54,7 +64,9 @@ struct FileRepository {
   }
 
   gcc_pure
-  const AvailableFile *FindByName(const char *name) const;
+  const AvailableFile *FindByName(std::string name) const;
```

Comment:
Again, unnecessary bloat - and you even changed an existing lean interface to a bloated one, for no reason.

Reply:
I tried to be consistent with using `std::string` as identifiers for the files.
But yeah they should references.

Reply:
Why references? That's still bloat, just slightly less.